ckanext/depositar_theme/templates/home/index/idx_stats.html

```diff
@@ -0,0 +1,40 @@
+{% set formatCntList = h.get_format_count() %}
+
+<div class="idx stats">
+  <div class="container">
+    <div class="stat-data">
+      <div class="imp-data">
+        <div class="imp-item">
+          <h1>{{ _('Datasets') }}</h1>
+          <a href="{{ h.url_for('dataset.search') }}" class="btn">
+            <p class="cnt">{{ h.get_site_statistics()['dataset_count'] }}</p>
+          </a>
+        </div>
+        <div class="imp-item">
+          <h1>{{ _('Projects') }}</h1>
+          <a href="{{ h.url_for('organization.index') }}" class="btn">
+            <p class="cnt">{{ h.get_site_statistics()['organization_count'] }}</p>
+          </a>
+        </div>
+        <div class="imp-item" href="#" data-toggle="tooltip" data-placement="top" title="{{ _('Total views of all datasets since 2022-03-25.') }}">
```

Comment:
```suggestion
        <div class="imp-item" data-toggle="tooltip" data-placement="top" title="{{ _('Total views of all datasets since 2022-03-25.') }}">
```

另外，tooltip 沒有正確顯示。請依照[官方文件](https://getbootstrap.com/docs/4.6/components/tooltips/)加入所需的 Popper library，再以 js 觸發 tooltip 事件。正確效果如下：

<img width="273" alt="截圖 2022-06-14 下午2 45 26" src="https://user-images.githubusercontent.com/276250/173511728-98770ffd-43b0-4262-b8af-07cc4afd137a.png">